api/apps/articles/model/schema.coffee

```diff
@@ -77,6 +77,9 @@ denormalizedArtwork = (->
   thumbnail_teaser: @string().allow('', null)
   thumbnail_image: @string().allow('', null)
   tags: @array().items(@string()).default([])
+  vertical: @object().keys
+    name: @string().allow('').default('')
```

Comment:
Sorry one last thing, do we need a default here? If we open up and resave an old article, it's going to have a `vertical: name: ''` in the article model and I think that'd be confusing without the id in there as well. Maybe it'd be more clear if we let the object be `null` which means no vertical field will be saved in the first place.